docs/SETUP.md

```diff
@@ -33,28 +33,29 @@ The following instructions were modified from the [InvenioRDM System Requirement
 - Supported Operating Systems
   - MacOS or Linux-based systems (Windows not supported)
 - System Requirements to Install Ultraviolet (ensure you have these installed on your system)
-  - **Git**
-  - **Python Version Management (Pyenv)**
+  - **Version Control (Git 2.41.0+)**
+    - MacOS(arm) download full Xcode via the appstore (this will bring in development tools like Git, but will also setup for success when downloading Cairo). Check for installation with `xcode-select -p`. Ensure to open Xcode to accept license agreement.
```

Comment:
I think you mean that full Xcode  will help to install Cairo later not just download it. Correct ?